manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: Lib Nyanpasu Org

```

```suggestion
Publisher: elaina
```
This should match what is written to control panel; If it does not, `AppsAndFeaturesEntries` are required in the installer manifest

---

I'm not seeing any Publisher listed in the [Apps and features](https://github.com/microsoft/winget-pkgs/pull/148633#issuecomment-2052292193) page entry. This [manifest field](https://github.com/microsoft/winget-pkgs/blob/1d9936235b46cc3e4453c5f63e4489aace453890/manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.locale.en-US.yaml#L7) was updated to be eponymous, is that accurate?